tools/azsdk-cli/Azure.Sdk.Tools.Cli/Tools/SpecWorkflowTool/SpecWorkFlowTool.cs

```diff
@@ -55,7 +55,56 @@ public class SpecWorkflowTool(IGitHubService githubService,
         private readonly Option<int> releasePlanIdOpt = new(["--release-plan"], "SDK release plan id") { IsRequired = false };
         private readonly Option<int> workItemOptionalIdOpt = new(["--workitem-id"], "Release plan work item id") { IsRequired = false };
 
+        private async Task<GenericResponse> IsReleasePlanReadyToGenerateSDKAsync(int workItemId, string language)
```

Comment:
Generic question, for all the messages added to `response.Details` is the expectation that with that the agent takes action? or the user?
My concern is that we are exposing to the user that there is a work item per Release Plan, but they don't know that and they shouldn't. They just interact with the Release plan. So if the message goes to the users, it might be better to use the release plan ID in the messages instead of the workitem id

Reply:
It's for the agent to take action and provide correct value back when invoking MCP tool Challenge of using release Plan ID in all MCP tool call is that it's not unique and agent has no idea which environment it needs to connect.  If a release plan is used, it is always checked in production. User can still provide a release plan Id in the prompt and agent find the corresponding work item and uses it in further calls. We don't need user to know the work item Id.